### PR TITLE
Feat/user destroy

### PR DESCRIPTION
**Pull request by leodhb**
# Description
user deletion (anonymize) mechanism using jwt

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [ ] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Test A

### CODE REVIEW

File: app/commands/users/anonymize.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    def call
+      with_exception_handle do
+        user.update!(email: dummy_email)
+      end
+    end
+
+    private
+
+    def dummy_email
+      "deleted_user+#{SecureRandom.hex(10)}@ribon.io"
```
Comment (thaisribon) [inline app/commands/users/anonymize.rb:22]: we could use the user id here: "deleted_user+#{user.id}@ribon.io", because it will always be unique

Comment (leodhb) [reply app/commands/users/anonymize.rb]: Nice! Done =)

File: app/jobs/mailers/send_user_deletion_email_job.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    def send_email(user, jwt)
+      SendgridWebMailer.send_email(receiver: user.email,
+                                   dynamic_template_data: { url: url(jwt) },
+                                   template_name: 'user_account_deletion_id',
+                                   language: user.language).deliver_later
+    end
+
+    def url(jwt)
+      "https://dapp.ribon.io/delete_account?token=#{jwt}"
```
Comment (thaisribon) [inline app/jobs/mailers/send_user_deletion_email_job.rb:19]: does this url exists?

Comment (leodhb) [reply app/jobs/mailers/send_user_deletion_email_job.rb:19]: Will exist in our web client. This is just the first part

Comment (thaisribon) [reply app/jobs/mailers/send_user_deletion_email_job.rb:19]: ok :)

**Review by thaisribon — APPROVED** [review]

**Pull request merged**